test/colony.js

```diff
@@ -87,6 +89,11 @@ contract("Colony", accounts => {
     const clnyToken = await Token.new("Colony Network Token", "CLNY", 18);
     await colonyNetwork.createMetaColony(clnyToken.address);
 
+    const reputationMiningCycle = await ReputationMiningCycle.new();
+    const reputationMiningCycleRespond = await ReputationMiningCycleRespond.new();
+    const reputationMiningCycleResolver = await Resolver.new();
+    await setupReputationVersionResolver(reputationMiningCycle, reputationMiningCycleRespond, reputationMiningCycleResolver, colonyNetwork);
```

Comment:
There's no need to wire this up again, we've done it already in migrations and can use the existing `miningCycleResolver`.